pydevices/HtsDevices/acq2106_435st.py

```diff
@@ -357,9 +357,14 @@ def run(self):
                         while toread:
```

Comment:
this line seems different from line 299 of the other device `while toread and self.running:`.
given that the complexity of the run method is quite high it would probably be best to share one implementation, where possible.